packages/web/src/components/SequenceView/SequenceMarkerMutation.tsx

```diff
@@ -56,7 +56,8 @@ function SequenceMarkerMutationDisconnected({
 
   const fill = getNucleotideColor(queryNuc)
   const width = Math.max(BASE_MIN_WIDTH_PX, pixelsPerBase)
-  const x = pos * pixelsPerBase - width / 2
+  const halfNuc = Math.max(pixelsPerBase, BASE_MIN_WIDTH_PX) / 2 // Anchor on the center of the first nuc
+  const x = pos * pixelsPerBase - halfNuc
```

Comment:
This is the only non-range marker changed: the nuc substitution marker.

Similarly, I shift it to the left by half a nuc. Note it was already shifted before by (width / 2). But because width is always 1 nuc, I find it more explicit to subtract the half a nuc. Gives the same result, but looks more in line with the rest of markers.